packages/actions/src/Concerns/CanImportRecords.php

```diff
@@ -140,7 +142,8 @@ protected function setUp(): void
                         $csvReader->setDelimiter($csvDelimiter);
                     }
 
-                    $csvReader->setHeaderOffset(0);
+                    $headerOffset = $this->getHeaderOffset();
+                    $csvReader->setHeaderOffset($headerOffset);
```

Comment:
$headerOffset isn't used, wouldn't it make more sense to:
```php
$csvReader->setHeaderOffset($this->getHeaderOffset());
```

Reply:
I had that initially but if you look at the code directly below this:

```php
$csvColumns = $csvReader->getHeader();
$csvColumnOptions = array_combine($csvColumns, $csvColumns);
```

My code mirrors this styling.